From 93f7f876ca345e186e0990ab35defa0eebe54e80 Mon Sep 17 00:00:00 2001
From: Oleksii Gulchenko <oleksii.gulchenko@globallogic.com>
Date: Tue, 21 May 2019 16:57:05 +0300
Subject: [PATCH] W/A: Not forbidden executables is allowed by default

This workaround created for applications which not listed in
list of forbidden apps.

Signed-off-by: Oleksii Gulchenko <oleksii.gulchenko@globallogic.com>
Change-Id: Ief802a886de72413f67621350811078451cae8b3
(cherry picked from commit fe62ac7477070cb103df21f17547c88bbc809224)
---
 ui/build/paths/config.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ui/build/paths/config.go b/ui/build/paths/config.go
index 571740133..8852f7d5f 100644
--- a/ui/build/paths/config.go
+++ b/ui/build/paths/config.go
@@ -56,7 +56,7 @@ var Log = PathConfig{
 var Missing = PathConfig{
 	Symlink: true,
 	Log:     true,
-	Error:   true,
+	Error:   false,
 }
 
 var LinuxOnlyPrebuilt = PathConfig{
-- 
2.28.0

